azure-pipelines.yml

```diff
@@ -216,7 +216,7 @@ jobs:
       --test_output=errors \
       --define DISABLE_VERILATOR_BUILD=true \
       --build_tag_filters=-vivado \
-      --test_tag_filters=-broken,-cw310,-verilator,-dv \
+      --test_tag_filters=-broken,-cw310,-verilator,-dv,-vivado \
```

Comment:
Looks like we were leaning on `-cw310` in the test tag filters. I verified locally that we need `-vivado` in both the build tag filters and the test tag filters to prevent `updatemem` from being invoked.

Reply:
Thanks I missed that yesterday